contracts/NodeDelegator.sol

```diff
@@ -64,26 +66,61 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
     }
 
     /// @notice Deposits an asset lying in this NDC into its strategy
-    /// @dev only supported assets can be deposited and only called by the LRT manager
+    /// @dev only supported assets can be deposited and only called by the LRT Operator
     /// @param asset the asset to deposit
     function depositAssetIntoStrategy(address asset)
         external
         override
         whenNotPaused
         nonReentrant
         onlySupportedAsset(asset)
-        onlyLRTManager
+        onlyLRTOperator
+    {
+        _depositAssetIntoStrategy(asset);
+    }
+
+    /// @notice Deposits all specified assets lying in this NDC into its strategy
+    /// @dev only supported assets can be deposited and only called by the LRT Operator
+    /// @param assets List of assets to deposit
+    function depositAssetsIntoStrategy(address[] calldata assets)
+        external
+        override
+        whenNotPaused
+        nonReentrant
+        onlyLRTOperator
     {
+        // For each of the specified assets
+        for (uint256 i; i < assets.length;) {
+            // Check the asset is supported
+            if (!lrtConfig.isSupportedAsset(assets[i])) {
+                revert ILRTConfig.AssetNotSupported();
+            }
+
+            _depositAssetIntoStrategy(assets[i]);
+
+            unchecked {
+                ++i;
+            }
+        }
+    }
+
+    function _depositAssetIntoStrategy(address asset) internal {
```

Comment:
Let's add a comment to this that the caller is responsible for verifying that the asset is supported.